_data/changelogs/memorable-date.yml

```diff
@@ -3,14 +3,29 @@ type: component
 changelogURL:
 items:
 - date: 2022-11-14
-  summary: Updated the month entry to a `select` from an `input`. 
-  summaryAdditional: Recent usability testing indicated that using a `select` for month entry was more clear and reduced mistakes. 
+  summary: Updated the month entry to a `select` from an `input`.
+  summaryAdditional: Recent usability testing indicated that using a `select` for month entry was more clear and reduced mistakes.
   affectsMarkup: true
   affectsStyles: true
   affectsGuidance: true
   githubRepo: uswds
   githubPr: 4937
   versionUswds: 3.3.0
+- date: 2022-04-28
+  summary: Added guidance for adding labels and using text boxes in place of select elements.
+  affectsGuidance: true
+  githubPr: 1538
+  githubRepo: uswds-site
+- date: 2022-04-28
+  summary: Updated package name to `usa-memorable-date`.
+  affectsGuidance: true
+  githubPr: 1538
+  githubRepo: uswds-site
+- date: 2022-04-28
+  summary: Renamed component from "date input" to "memorable date".
```

Comment:
```suggestion
  summary: Renamed component from `date input` to `memorable date`.
```

Reply:
As discussed, going to leave these items with double quotes because it is referencing the name of the component, not just the package.